lang/cs/Org.Apache.REEF.Network/Group/Operators/Impl/BroadcastOperatorSpec.cs

```diff
@@ -35,9 +36,32 @@ public BroadcastOperatorSpec(string senderId, ICodec<T> codecType)
         {
             SenderId = senderId;
             Codec = codecType;
+            PipelineDataConverter = null;
```

Comment:
For reducer spec, you set DefaultPipelineDataConverte if PipelineDataConverter is not provided, here null is used. Any difference?